tool/tool.iml

```diff
@@ -34,6 +35,15 @@
     <content url="file://$MODULE_DIR$/../../antlr4-csharp/tool/test">
       <sourceFolder url="file://$MODULE_DIR$/../../antlr4-csharp/tool/test" isTestSource="false" />
     </content>
+    <content url="file://$MODULE_DIR$/../../antlr4-javascript">
+      <sourceFolder url="file://$MODULE_DIR$/../../antlr4-javascript/tool/src" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/../../antlr4-javascript/tool/test" isTestSource="false" />
+      <excludeFolder url="file://$MODULE_DIR$/../../antlr4-javascript/tool/test/org/antlr/v4/test/rt" />
+      <excludeFolder url="file://$MODULE_DIR$/../../antlr4-javascript/tool/test/org/antlr/v4/test/runtime/javascript/chrome" />
```

Comment:
:question: Are these excludes needed (the browsers, from `chrome` to `safari` below)? It was my understanding that generation for these folders is currently disabled.